operations/network/customer_accessible_networks/can_to_chn/network/chn_enable.md

```diff
@@ -547,10 +565,19 @@ CHN network configuration of compute nodes is performed by the UAN CFS configura
 
 1. (`ncn-m001#`) Enable CFS changes on compute nodes.
 
-   ```bash
+    ```bash
    for xname in $(cray hsm state components list --role Compute --type Node --format json | jq -r .Components[].ID) ; do
       cray cfs v3 components update --enabled true --state "[]" --format json "${xname}"
    done
+    ```
+
+    Alternatively, enable multiple components at once by using the `updatemany` command.
+    Refer to the [Managing many components](../../../../configuration_management/CFS_Commands_Cheat_Sheet.md#managing-many-components) for `updatemany` command requirements.
```

Comment:
````suggestion
   ```

   Alternatively, if running CSM 1.6.2 or higher, enable multiple components at once by using the `updatemany` command. Refer to
   [Managing many components](../../../../configuration_management/CFS_Commands_Cheat_Sheet.md#managing-many-components)
   for complete `updatemany` command requirements and usage.
````